Sources/SpeziAccount/ViewModifier/AccountRequiredModifier.swift

```diff
@@ -32,15 +32,19 @@ struct AccountRequiredModifier<SetupSheet: View>: ViewModifier {
             return true // not signedIn
         }
 
-        // we present the sheet if the account is anonymous and we do not consider anonymous accounts to the fully signed in
-        return details.isAnonymous && !considerAnonymousAccounts
+        // we present the sheet if the account is not valid (yet)
+        return !isValid(details)
     }
 
 
-    init(enabled: Bool, considerAnonymousAccounts: Bool, @ViewBuilder setupSheet: () -> SetupSheet) {
+    init(
+        enabled: Bool,
+        isValid: @escaping (AccountDetails) -> Bool,
```

Comment:
I think we might want to rename this a bit to make it clear what this closure does; maybe something like `accountSetupIsComplete` or something similar?

Reply:
renamed it to `accountSetupIsComplete `